charts/generic/generic.chartserviceversion.yaml

```diff
@@ -41,7 +41,7 @@ spec:
     - name: Kubernetes Slack
       url: https://slack.kubernetes.io/ 
     - name: Documentation
-      url: https://kubernetes.io/docs/home/
+      url: https://docs.litmuschaos.io/docs/chaoshub/#generic-chaos
   icon:
```

Comment:
If we are providing litmus-docs link in Documentation then I think Kubernetes slack channel link should also changed to litmus channel.
Or we can leave it like that only i.e Kubernetes docs in documentation because we have litmus-docs link inside individual chart of experiments. @ajeshbaby 
cc: @ksatchit

Reply:
Since we are using Kubernetes slack we can retain the same.